matching-service/model/match-model.js

```diff
@@ -0,0 +1,21 @@
+import mongoose from "mongoose";
+var Schema = mongoose.Schema;
+let MatchModelSchema = new Schema({
+    matchId: {
```

Comment:
Since I am passing you the difficulty in the socket.emit args, do you think it is better to also store the difficulty somewhere ?